src/views/PopupView/ImportLabelPopup/ImportLabelPopup.tsx

```diff
@@ -30,64 +29,69 @@ const ImportLabelPopup: React.FC<IProps> = (
         updateActiveLabelTypeAction
     }) => {
     const resolveFormatType = (labelType: LabelType): AnnotationFormatType => {
-        const possibleImportFormats = ImportFormatData[labelType]
-        return possibleImportFormats.length === 1 ? possibleImportFormats[0].type : null
-    }
+        const possibleImportFormats = ImportFormatData[labelType];
+        return possibleImportFormats.length === 1 ? possibleImportFormats[0].type : null;
+    };
 
     const [labelType, setLabelType] = useState(activeLabelType);
     const [formatType, setFormatType] = useState(resolveFormatType(activeLabelType));
     const [loadedLabelNames, setLoadedLabelNames] = useState([]);
     const [loadedImageData, setLoadedImageData] = useState([]);
     const [annotationsLoadedError, setAnnotationsLoadedError] = useState(null);
 
-    const {getRootProps, getInputProps} = useDropzone({
-        accept: [AcceptedFileType.JSON, AcceptedFileType.TEXT],
-        multiple: true,
-        onDrop: (acceptedFiles) => {
-            const importer = new (ImporterSpecData[formatType])([labelType])
-            importer.import(acceptedFiles, onAnnotationLoadSuccess, onAnnotationsLoadFailure);
-        }
-    });
 
-    const onLabelTypeChange = (labelType: LabelType) => {
-        setLabelType(labelType);
-        setFormatType(resolveFormatType(labelType))
+
+    const onLabelTypeChange = (type: LabelType) => {
+        setLabelType(type);
+        setFormatType(resolveFormatType(type));
         setLoadedLabelNames([]);
         setLoadedImageData([]);
         setAnnotationsLoadedError(null);
-    }
+    };
 
     const onAnnotationLoadSuccess = (imagesData: ImageData[], labelNames: LabelName[]) => {
         setLoadedLabelNames(labelNames);
         setLoadedImageData(imagesData);
         setAnnotationsLoadedError(null);
-    }
+    };
 
-    const onAnnotationsLoadFailure = (error?:Error) => {
+    const onAnnotationsLoadFailure = (error?: Error) => {
         setLoadedLabelNames([]);
         setLoadedImageData([]);
         setAnnotationsLoadedError(error);
     };
 
-    const onAccept = (labelType: LabelType) => {
+    const { getRootProps, getInputProps } = useDropzone({
```

Comment:
I tested loading COCO JSON, and I was not able to select `.json` file from the disk. It was grayed out.

Reply:
I guess it's because the new dropzone filetype api

Reply:
fixed.